bin/mkdotenv

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+# !/bin/sh
```

Comment:
Looks like those shebang problems are here, too. Maybe you want to rebase or merge in your other branch?